provider/pkg/provider/helm_release.go

```diff
@@ -250,11 +249,37 @@ func (r *helmReleaseProvider) getActionConfig(namespace string) (*action.Configu
 
 func decodeRelease(pm resource.PropertyMap) (*Release, error) {
 	var release Release
+	values := map[string]interface{}{}
 	stripped := pm.MapRepl(nil, mapReplStripSecrets)
 	logger.V(9).Infof("Decoding release: %#v", stripped)
+
+	if pm.HasValue("valueYamlFiles") {
+		v := stripped["valueYamlFiles"]
+		switch reflect.TypeOf(v).Kind() {
+		case reflect.Slice, reflect.Array:
+			s := reflect.ValueOf(v)
+			for i := 0; i < s.Len(); i++ {
+				val := s.Index(i).Interface()
+				switch t := val.(type) {
+				case *resource.Asset:
+					b, err := t.Bytes()
+					if err != nil {
+						return nil, err
+					}
+					if err = yaml.Unmarshal(b, &values); err != nil {
+						return nil, err
+					}
+				default:
+					return nil, fmt.Errorf("unsupported type for 'valueYamlFiles' arg: %T", v)
```

Comment:
Do we want to also support Archives? I just noticed that the SDK type supports both Assets and Archives.

Reply:
Trying to keep this in-line with the helm CLI. The CLI takes yaml files and not archives. We can reconsider if we get requests from users. I can't think of an obvious use case where archives would be preferred.